src/server/_query.py

```diff
@@ -499,7 +546,14 @@ def to_asc(v: Union[str, bool]) -> str:
                 return "DESC"
             return cast(str, v)
 
-        args_order = [f"{self.alias}.{k} ASC" for k in args]
+        # Use the column name without their table name for the Order By clause since Union All is used
+        args_order = []
+        for i in range(len(args)):
+            # avoid conflict with the reserved word
+            if args[i] == 'signal':
```

Comment:
alas, we have more columns in epidata that use reserved words than just `signal` -- is there a downside to escaping all the columns? e.g.
```
args_order = [f"`{k}` ASC" for k in args]
```

Reply:
Edited √